wbooks/src/components/book-suggestions/index.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+import BookSuggestions from './layout';
+
+class BookSuggestionsContainer extends Component {
```

Comment:
keep in mind taht if your component is just dumb (no need states and functions) just use the index.js file